tests/database/test_model.py

```diff
@@ -68,7 +83,11 @@ class TestSession(ModelTestBase):
     klass = model.Session
 
     def _db_obj_get_dependencies(self):
-        tenant = model.Tenant(name="My Other Tenant", ssh_key="my other public SSH key")
+        tenant = model.Tenant(
+            name="My Other Tenant",
+            api_key=str(uuid4()),
```

Comment:
`api_key` should accept `UUID` objects, there should be no need to cast it to `str` here and below.

Reply:
Latest push fixes this.